cmd/meroxa/root/resources/create.go

```diff
@@ -153,7 +153,9 @@ func (c *Create) Execute(ctx context.Context) error {
 		Metadata: nil,
 	}
 
-	if c.flags.Type != string(meroxa.ResourceTypeNotion) && c.flags.URL == "" {
+	if (c.flags.Type != string(meroxa.ResourceTypeNotion) &&
+		c.flags.Type != string(meroxa.ResourceTypeSpireMaritimeAIS)) &&
```

Comment:
Is it possible to add a unit test case for that?

Reply:
sure thing